pkg/pgmodel/migrate.go

```
@@ -223,9 +239,19 @@ func ensureVersionTable(db *pgxpool.Pool) error {
 	return nil
 }
 
-func getDBVersion(db *pgxpool.Pool) (semver.Version, error) {
+func getSchemaVersion(db *pgxpool.Pool) (semver.Version, error) {
+	var version semver.Version
```

Sorry, but I didn't get where are we initializing this.

---

Looks like this is just a placeholder value to send when an error occurs that should be ignored.

---

Yeah, it's pretty silly. In my defense, it was like that when I found it 😉